### DISCUSSION TITLE
Welcome to cs-case-sim Discussions!

### DISCUSSION THREAD

**Discussion by jmohan57** [jmohan57/cs-case-sim#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmohan57** [comment]
![Uploading image.png…]()

**Comment by venomsnake0777** [comment]
![image](https://github.com/jmohan57/cs-case-sim/assets/170359319/adcb2da9-0f85-4a8f-9cd6-a20ff755df94)
this commit is expried

**Reply by jmohan57** [reply]
> ![image](https://private-user-images.githubusercontent.com/170359319/336953823-adcb2da9-0f85-4a8f-9cd6-a20ff755df94.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxODY5ODksIm5iZiI6MTcxODE4NjY4OSwicGF0aCI6Ii8xNzAzNTkzMTkvMzM2OTUzODIzLWFkY2IyZGE5LTBmODUtNGE4Zi05Y2Q2LWEyMGZmNzU1ZGY5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMlQxMDA0NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iM2ExMGQwZTJmOTk2NTU4OWFlMWI5NmFmNDQzMGVlOGVlZjU2NzI1MDNmNzRjN2IwOGEzODFiYmY2MjRlMTEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.FLPy2oYf1fKqvkb2HZ5LnkucwM3Fzv6wzObTRek8Rbc) this commit is expried

what's up?

**Comment by venomsnake0777** [comment]
_**How can I fix it?**_

**Comment by SecretWai** [comment]
Hello Nice to meet you.

**Reply by jmohan57** [reply]
> Hello Nice to meet you.

fine, Nice to meet you too.

**Reply by SecretWai** [reply]
> > Hello Nice to meet you.
> 
> fine, Nice to meet you too.

Hey, can you explain your projects?

**Comment by jmohan57** [comment]
Did you read requirement of project? 🤨
